### PR TITLE
NFDIV-4299: Add fortify scans to nightly pipeline

### PR DESCRIPTION
**Pull request by FaisalMoJ**
### Change description ###

Add fortify scans to nightly pipeline.

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-4299

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: config/fortify-client.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+fortify.client.releaseId=134905
```
Comment (adamg-hmcts) [inline config/fortify-client.properties:1]: I think the release ID for frontend is 134904 (different to case-api)
```suggestion
fortify.client.releaseId=134904
```

File: package.json [hunk 2]
```diff
@@ -36,7 +36,8 @@
     "test:crossbrowser:playwright:docker": "docker pull mcr.microsoft.com/playwright:focal && docker run --rm --volume $PWD:/home/app --workdir='/home/app' --env TEST_PASSWORD --env TEST_URL mcr.microsoft.com/playwright:focal bash -x src/test/docker-yarn.sh test:crossbrowser:playwright $UID",
     "test:crossbrowser:sauce": "SAUCE='true' yarn test:crossbrowser:codeceptjs run --grep '^(?!.*@flaky)'",
     "test:crossbrowser:codeceptjs": "codeceptjs run --config ./src/test/cross-browser/codecept.conf.js",
-    "cichecks": "yarn && run-p build lint test"
+    "cichecks": "yarn && run-p build lint test",
+    "fortifyScan": "./test/java/gradlew -p test/java fortifyScan"
```
Comment (adamg-hmcts) [inline package.json:40]: We may need to include `src` in the paths?

File: src/test/java/.gitignore [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+.idea/
+*.iws
+*.iml
+*.ipr
+
+### Gradle
+.gradle/
+out/
+bin/
+build/
```
Comment (adamg-hmcts) [inline src/test/java/.gitignore:17]: If there are some new exclusions here, maybe we could add them to the .gitignore we have at the root of the project and delete this second gitignore file?

Comment (FaisalMoJ) [reply src/test/java/.gitignore:17]: That's a good idea thanks i will compare this with the one we have and add the missing bits 👍

**Review by adamg-hmcts — APPROVED** [review]

**Pull request merged**